### ISSUE TITLE
Breadcrumb Accessibility Tab updates and DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon-website#573]
This issue is related to #491 
The following updates are needed to the Breadcrumb Accessibility Tab:
**1. Remove the first paragraph:** 
"The Breadcrumb React Carbon component has been tested against the latest W3C Web Content Accessibility Guidelines (WCAG) 2.1 Level A and AA success criteria and violations have been identified as high priority issues. This document will be updated when these accessibility issues are resolved."

**Note:** This paragraph is part of the Non-compliant template and should not be added to the completed documents. The non-compliant template is used when the document can not be completed due to open accessibility issues that block completion.

**2. Under the Heading "Accessibility Testing" add the following paragraph:**
Accessibility issues are tracked in the [Carbon Component GitHub repository](https://github.com/carbon-design-system/carbon/milestone/57).

**3. The following DAP violations are also found. Let me know if they need to be reported on a separate issue (see screenshot below):**
<img width="1230" alt="Screen Shot 2019-11-11 at 2 17 03 PM" src="https://user-images.githubusercontent.com/21676914/68618227-7e5b4800-048e-11ea-842f-433d259f8d1c.png">
